src/UIOverlay.py

```diff
@@ -451,14 +455,21 @@ def _update_shopping_list_mode(self) -> None:
         self._save_config()
 
     def _update_shopping_list(self) -> None:
-        # TODO: validate it
         shopping_list = list(map(lambda x: x.strip(), self._shopping_list_entry.get().split(",")))
+        if len(shopping_list) == 0 or len(self._shopping_list_entry.get().strip()) == 0:
+            self._update_shopping_list_label("Error: Must enter at least one item")
+            raise ValueError
         for item in shopping_list:
             if item not in self._items_map.items():
-                print("Unknown Recipe:", item)
+                self._update_shopping_list_label('Error: unknown item "{0}"'.format(item))
                 raise ValueError
```

Comment:
I think you could simple return from this function here instead of raising an exception.